### DISCUSSION TITLE
Does this cache games that are downloaded frequently for high speed downloads?

### DISCUSSION THREAD

**Discussion by Alfredoes234** [Phalcode/gamevault-backend#307] [Q&A] [answered]
I was just wandering if this was a thing, somewhat similar to lan-cache or the like allowing for really fast downloads.

**Comment by Alfagun74** [comment]
No, and based on your description I can't think of any technical implementation. What do you mean?

**Reply by Alfredoes234** [reply]
I might be Im being really dumb but what lancache does, is it intercepts (DNS) http requests for downloading games and caches them, this allows for speed up compared to downloading from the same source. It is commonly used for Lan events. I have a feeling this would not matter then based on your reaction due to self hosting.

**Reply by Alfagun74** [reply, marked as answer]
I see. No we do not support such functionality. you can simply connect to the server directly via IP though if it is in your local area.
